<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMonthlyOccPerc</class>
 <widget class="QWidget" name="FMonthlyOccPerc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EYearCombo" name="cbYear"/>
   </item>
   <item>
    <widget class="EDateEdit" name="deDate1"/>
   </item>
   <item>
    <widget class="EDateEdit" name="deDate2"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chGroupByDays">
     <property name="text">
      <string>Group by days</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>611</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EYearCombo</class>
   <extends>QComboBox</extends>
   <header location="global">eyearcombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
